research-software-directory/RSD-as-a-service
# feat: update edit section ui components

# Update ui 

Closes #1099
Closes #1126

Changes proposed in this pull request:
* Use primary button (contained variant) in edit sections
* Use outlined inputs in edit sections
* Remove helper text (not adding a value and make layout more cluttered than needed)
* Changed order in chips input (first selection then selected items)
* Moved ConceptDOI input from left panel to central panel (edit software section). Validate DOI button will automatically update version DOI to concept DOI if possible.
* Update broken tests
* Split edit basic software information into description and links & metadata pages.

How to test:
* `make start` to build app and generate data 
* Add new software and validate working of edit software section
* Check working of validateDOI button
   * use version DOI 10.5281/zenodo.6575742
   * use concept DOI  10.5281/zenodo.6379973
* Add new project and validate working of edit project section
* Login as rsd-admin. Visit organisation and validate working of the edit organisation settings.
* Visit rsd-admin section and validate working of the admin section

## Edit software

- Description
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/f1272acc-1031-43a3-b16e-ee80c71097c1)

- Links & metadata
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/0ff42660-74d3-4fbb-9488-787d2f6b4176)

- Contributors
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/10f6770e-fd21-45d0-857d-45f1ad883929)

- Organisations
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/3bc27e9b-4cb7-4f56-8502-2ca23e9bcbb8)

- Mentions
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/fca6ebb5-58ad-49ac-98e6-ee93f618f459)

- Mantainers
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/33cd6061-861d-4b22-a2b2-fe87c026f515)

## Edit project
- Details
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/b899f04c-6e4a-4a62-9c78-8aeaa978a19a)

- Mentions
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/359eeb0c-c4e2-4d3f-929c-54c29cf1ddcd)


## Organisation
- Settings
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/64d43636-ef94-489c-b879-06b050126093)

## RSD admin section
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/9e36d3be-63e0-4235-860a-1d715d6436ee)


PR Checklist:
